diff --git a/node_modules/@abgov/jsonforms-components/index.esm.js b/node_modules/@abgov/jsonforms-components/index.esm.js
index 62bcb07..f0bd6a0 100644
--- a/node_modules/@abgov/jsonforms-components/index.esm.js
+++ b/node_modules/@abgov/jsonforms-components/index.esm.js
@@ -3,13 +3,15 @@ import React, { createContext, useContext, useReducer, useMemo, useEffect, useSt
 import { GoAFormItem, GoAInput, GoATextArea, GoACallout, GoAInputDate, GoAInputDateTime, GoAInputTime, GoARadioGroup, GoARadioItem, GoACheckbox, GoAGrid, GoAFormStepper, GoAFormStep, GoAPages, GoAButton, GoAModal, GoAButtonGroup, GoAIconButton, GoAFileUploadInput, GoACircularProgress, GoAContainer, GoATable, GoADropdown, GoADropdownItem, GoADetails, GoASkeleton } from '@abgov/react-components';
 import styled from 'styled-components';
 import axios from 'axios';
-import get$1 from 'lodash/get';
+import get$1 from 'lodash/get.js';
 import { rankWith, isStringControl, and, optionIs, uiTypeIs, isDateControl, isNumberControl, isIntegerControl, isDateTimeControl, isTimeControl, isEnumControl, isBooleanControl, getAjv, isVisible, isEnabled, deriveLabelForUISchemaElement, schemaTypeIs, formatIs, createDefaultValue, Paths, or, isObjectArrayControl, isPrimitiveArrayControl, isControl, isScoped, hasType, isCategorization, isLayout } from '@jsonforms/core';
 import { withJsonFormsControlProps, withJsonFormsRendererProps, withJsonFormsEnumProps, withTranslateProps, useJsonForms, JsonFormsDispatch, withJsonFormsLayoutProps, withJsonFormsArrayLayoutProps, withJsonFormsCellProps } from '@jsonforms/react';
-import { isEqual, isObject as isObject$e } from 'lodash';
-import merge from 'lodash/merge';
-import isEmpty$1 from 'lodash/isEmpty';
-import range from 'lodash/range';
+// import { isEqual, isObject as isObject$e } from 'lodash';
+import isEqual from 'lodash/isEqual.js';
+import isObject$e from 'lodash/isObject.js';
+import merge from 'lodash/merge.js';
+import isEmpty$1 from 'lodash/isEmpty.js';
+import range from 'lodash/range.js';
 import Ajv from 'ajv';
 import addErrors from 'ajv-errors';
 import addFormats from 'ajv-formats';
@@ -7389,7 +7391,7 @@ const ListWithDetailsControl = props => {
       open: open,
       onCancel: deleteCancel,
       onConfirm: deleteConfirm,
-      title: props.translations.deleteDialogTitle || '',
+      title: (props.translations && props.translations.deleteDialogTitle) || '',
       message: `Are you sure you wish to delete ${name} `
     })]
   });
